From 05cc09f8d492b0a85ecf92eb8b2fc6f64f1a3e7c Mon Sep 17 00:00:00 2001
From: Zhang Zekun <zhangzekun11@huawei.com>
Date: Mon, 20 Feb 2023 07:41:19 +0000
Subject: [PATCH 052/500] [Huawei] mm: share_pool: Add copyright head for
 mm/share_pool_internal.h

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 188411

------------------------------------

Add copy right header for mm/share_pool_internal.h.

Signed-off-by: Zhang Zekun <zhangzekun11@huawei.com>
---
 mm/share_pool_internal.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/mm/share_pool_internal.h b/mm/share_pool_internal.h
index 222993618dca..74ebf4739e2e 100644
--- a/mm/share_pool_internal.h
+++ b/mm/share_pool_internal.h
@@ -1,4 +1,8 @@
 /* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * Copyright (C) Huawei Technologies Co., Ltd. 2023. All rights reserved.
+ */
+
 #ifndef SHARE_POOL_INTERNAL_H
 #define SHARE_POOL_INTERNAL_H
 
-- 
2.17.1

